analysis/warlock/src/DemonicCirclesCreated.tsx

```diff
@@ -0,0 +1,38 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CreateEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+class DemonicCirclesCreated extends Analyzer {
+  static dependencies = {};
+
+  created = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.addEventListener(
+      Events.create.by(SELECTED_PLAYER).spell(SPELLS.DEMONIC_CIRCLE_SUMMON),
+      this.handleCreateEvent,
+    );
+  }
+
+  handleCreateEvent(event: CreateEvent) {
+    console.log('Handling create event', event);
```

Comment:
Probably don't need this log statement... if you want it put it behind a debugg check
Gotta keep that production console log clean

Reply:
Whoops, my bad, will remove it.